### PR TITLE
Add tailwindcss-intellisense

### PR DESCRIPTION
**Pull request by heavenshell**
https://github.com/tailwindlabs/tailwindcss-intellisense

Kapture 2020-07-31 at 0 25 31

### CODE REVIEW

**Comment by heavenshell** [comment]
I don't have any windows pc, it would be very applicate to try windows.

File: settings.json [hunk 1]
```diff
@@ -303,6 +303,15 @@
       "requires": [
         "npm"
       ]
+    },
+    {
+      "command": "tailwindcss-intellisense",
+      "config": {
+        "refresh_pattern": "\\(/\\|\\k\\+\\)$"
```
Comment (heavenshell) [inline settings.json:310]: I should tweak refresh_pattern...

**Comment by mattn** [comment]
Thank you.

**Comment by MindTooth** [comment]
I hate to post this in an old PR, but `--stdio` is passed twice, one inside the script and one inside the `server_info->cmd`.  Is this by design, @heavenshell?

I'm asking as I'm preparing a new MR for pumping the version.

**Comment by mattn** [comment]
This is my fault. `-stdio` should be specified in `settings/*.vim`

**Comment by mattn** [comment]
Fixed in #397

**Pull request merged**